cypress/tests/core/volto-slate/09-block-slate-slashmenu.js

```diff
@@ -3,16 +3,12 @@ import { slateBeforeEach } from '../../../support/e2e';
 describe('SlashMenu Test: Shortcuts', () => {
   beforeEach(slateBeforeEach);
 
-  it('As editor I can create a Description block using the SlashMenu shortcut', function () {
-    // Use SlashMenu shortcut to create a Description block and type some text in it
-    cy.getSlateEditorAndType('/Desc').type('{enter}');
-    cy.get('.documentDescription').type('This is a description.');
-    cy.get('.documentDescription').type('{enter}');
-
-    // Save
+  it('As editor I can create a Table block using the SlashMenu shortcut', function () {
+    // Use SlashMenu shortcut to create a table block
+    cy.getSlateEditorAndType('/t').type('{enter}');
     cy.toolbarSave();
 
-    // then the page view should contain a Description with the right text in it
-    cy.get('.documentDescription').contains('This is a description.');
+    // then the page view should contain a table
+    cy.get('#page-document table').should('be.visible');
```

Comment:
Should this be a new test, instead of modifying an existing test?

Reply:
@stevepiercy Cypress tests are slow. I decided it made more sense to modify the existing test to cover more functionality (it's still verifying that the blocks were filtered correctly, but now also verifies that it does a case-insensitive match anywhere in the block title, not only a case-sensitive match at the beginning) than add another very similar test.

If someone is motivated to add unit tests that cover more specific cases, that would be fine, but I won't be able to spend more time on this now.

Reply:
That's good thinking.